style.css

```diff
@@ -0,0 +1,73 @@
+section {
+    margin:auto;    
+    width: 625px;
+    background-color: lightgray;
+    color: white;
+}
+
+article {
+    display: inline-block;
+    width: 625px;
+    font-family: 'Roboto', serif;
+    font-weight: 400;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border-bottom: 25px solid transparent;
+}
+
+h1 {
+    display: inline-block;
+    font-family: 'Merriweather', serif;
```

Comment:
This is a exception to the message above, if a heading or something else requires a different font family, you set it per element like here